### PR TITLE
Fix for resolve yate runtime object.

### PR DESCRIPTION
**Pull request by vs-kurkin**

### CODE REVIEW

File: templates/js.tmpl [hunk 1]
```diff
@@ -7,7 +7,7 @@
 // На первое время, шаблоны (и соответственно matcher) выводятся только на верхнем уровне.
 
 module
-    var yr = yr || require('yate/lib/runtime.js');
+    var yr = yr || window.yr || require('yate/lib/runtime.js');
```
Comment (mutantcornholio) [inline templates/js.tmpl]: global?

Comment (vs-kurkin) [reply templates/js.tmpl]: fixed

**Comment by mutantcornholio** [comment]
нулан

**Comment by vs-kurkin** [comment]
Вливаю

**Pull request merged**